src/lib/utils/translations.ts

```diff
@@ -168,10 +168,15 @@ const getRequiredNamespacesForPath = (relativePath: string) => {
     requiredNamespaces = [...requiredNamespaces, "page-wallets"]
   }
 
+  // TODO: Remove this when the page is translated
   if (path.startsWith("/layer-2/")) {
     primaryNamespace = "page-layer-2"
   }
```

Comment:
Just noting that as-is the `page-layer-2` namespace will be loaded on `/layer-2/learn/`; is this what we want? If not, I'd change this one to not use `startsWith()`